prisma/schema.prisma

```diff
@@ -10,58 +10,102 @@ generator client {
+  createdAt    DateTime     @default(now()) @map("created_at")
+  updatedAt    DateTime     @default(now()) @updatedAt @map("updated_at")
+  deletedAt    DateTime?    @map("deleted_at")
+  @@map("Users")
 }
-
 model Membership {
-  id          Int     @id @default(autoincrement())
-  userId      Int
-  accountId   Int
-  roleId      Int
-  isConfirmed Boolean
-  user        User    @relation(fields: [userId], references: [id])
-  role        Role    @relation(fields: [roleId], references: [id])
-  account     Account @relation(fields: [accountId], references: [id])
+  id          Int       @id @default(autoincrement())
+  userId      Int       @map("user_id")
+  accountId   Int       @map("account_id")
+  roleId      Int       @map("role_id")
+  isConfirmed Boolean   @map("is_confirmed")
+  user        User      @relation(fields: [userId], references: [id])
+  role        Role      @relation(fields: [roleId], references: [id])
+  account     Account   @relation(fields: [accountId], references: [id])
+  createdAt   DateTime  @default(now()) @map("created_at")
+  updatedAt   DateTime  @default(now()) @updatedAt @map("updated_at")
+  deletedAt   DateTime? @map("deleted_at")
+  @@map("Memberships")
 }
-
 model Role {
   id         Int          @id @default(autoincrement())
-  roleName   UserRoles
+  roleName   UserRoles    @map("role_name")
   Membership Membership[]
+  createdAt  DateTime     @default(now()) @map("created_at")
+  updatedAt  DateTime     @default(now()) @updatedAt @map("updated_at")
+  deletedAt  DateTime?    @map("deleted_at")
+  @@map("Roles")
 }
-
 model Account {
-  id          Int          @id @default(autoincrement())
-  name        String
-  description String
-  password    String
-  Membership  Membership[]
-  Secret      Secret[]
+  id             Int             @id @default(autoincrement())
+  name           String
+  description    String
+  password       String
+  Membership     Membership[]
+  AccountSecret  AccountSecret[]
+  createdAt      DateTime        @default(now()) @map("created_at")
+  updatedAt      DateTime        @default(now()) @updatedAt @map("updated_at")
+  deletedAt      DateTime?       @map("deleted_at")
+  accountGiver   SecretShare[]   @relation("accountGiver")
+  accountReciver SecretShare[]   @relation("accountReciver")
+  @@map("Accounts")
 }
-
 model Secret {
-  id          Int     @id @default(autoincrement())
-  name        String
-  value       String
-  description String
-  accountId   Int
-  account     Account @relation(fields: [accountId], references: [id])
+  id            Int             @id @default(autoincrement())
+  name          String
+  value         String
+  description   String
+  createdAt     DateTime        @default(now()) @map("created_at")
+  updatedAt     DateTime        @default(now()) @updatedAt @map("updated_at")
+  deletedAt     DateTime?       @map("deleted_at")
+  AccountSecret AccountSecret[]
+  SecretsShare  SecretShare[]
+  @@map("Secrets")
+}
```

Comment:
You can run **npx prisma format** to format the schema.prisma file. The command will take care of validation and formatting.